server/calendar/calendar.js

```diff
@@ -0,0 +1,61 @@
+var gcal = require('google-calendar');
+var calendarController = require('./calendar-controller');
+
+var calendar = {
+  addCalendarGig: function (user, gigParams) {
+    var accessToken = user.accessToken;
+    var calendar_id = user.calendar_id;
+    var calendarInfo = user.gigInformation
+    gcal(accessToken).gigs.quickAdd(calendar_id, gigInformation, function(err, data) {
+      if (err) {
+        callback({error:err});
```

Comment:
all of the callbacks in this file are undefined. did you intend to add callback as a param to all of these methods?